src/main/java/com/softserveinc/ita/utils/WindowTabUtil.java

```diff
@@ -0,0 +1,18 @@
+package com.softserveinc.ita.utils;
+
+import com.codeborne.selenide.SelenideElement;
+import lombok.experimental.UtilityClass;
+
+import static com.codeborne.selenide.Selenide.*;
+
+@UtilityClass
+public class WindowTabUtil {
+    public static String getTitleFromSecondTab(SelenideElement selector) {
```

Comment:
empty line after class declaration

Reply:
all operations with selectors should be only in page objects or element util

such class should provide separate operations for the browser, take a look at Max's PR, calling methods for switching between tabs from class, and calling in test from page method for getting title

Reply:
create common page object for playmarkets with methods getAndroidAppTitle and ios

Reply:
Max's pr is merged